apps/backoffice-v2/package.json

```diff
@@ -50,7 +50,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@ballerine/common": "0.7.30",
```

Comment:
you might need to run changeset again since tomer pushed a version

Reply:
+1